components/modal/basic/src/index.scss

```diff
@@ -1,5 +1,9 @@
 @import '~@schibstedspain/theme-basic/lib/index';
 
+body.is-modal-open {
```

Comment:
I decided to preppend directly to "body" tag, because we only want this class acts on body, and because it seems pretty generic (is-modal-open), to avoid colisions I think it's better.